src/enum/order.enum.ts

```diff
@@ -0,0 +1,19 @@
+export enum DayOfWeekEnum {
```

Comment:
As I know for public libraries better to use type instead of an enum

```ts
type DayOfWeekType = 'monday' | 'tuesday' ...
```

Reply:
this only makes sense if types are in a different repository.